src/cmd/cmd.cpp

```diff
@@ -391,10 +391,10 @@ CmdOptions parseOptions(const QStringList &app_args)
             options.unsyncedfolders = it.next();
         } else if (option == QLatin1String("--max-sync-retries") && !it.peekNext().startsWith(QLatin1String("-"))) {
             options.restartTimes = it.next().toInt();
-        } else if (option == QLatin1String("--uplimit") && !it.peekNext().startsWith(QLatin1String("-"))) {
-            options.uplimit = it.next().toInt() * 1000;
-        } else if (option == QLatin1String("--downlimit") && !it.peekNext().startsWith(QLatin1String("-"))) {
-            options.downlimit = it.next().toInt() * 1000;
+        } else if (option == QLatin1String("--uplimitKbPerSec") && !it.peekNext().startsWith(QLatin1String("-"))) {
```

Comment:
Please don't change the commandline arguments, the change would break existing scripts.